base_install_and_test.sh

```diff
@@ -129,6 +128,8 @@ check_continue "updating geoips" "install conda"
         echo "    which conda (point to geoips_dependencies/miniconda3): "`which conda`
         date -u
     fi
+    source ~/.bashrc
```

Comment:
@jsolbrig Will this work on mac?

Reply:
Yeah, this does work on Mac, so long as someone is using Bash.

I don't know how I missed your comments on this. Going through them now and evaluating if this can be closed.